### ISSUE TITLE
Enhancement: provide a datum parameter

### ISSUE THREAD

**Issue by wmheigl** [geodynamics/sw4#100]
In borehole seismic applications, one often has velocity information over certain depth interval only and one is generally not interested in surface phenomena. It would be advantageous to have a datum parameter that allows to have the top of the computational grid not at z=0 but at some chosen depth, thus reducing the size of the computational grid and the computing time.

**Comment by andersp** [comment]
Thanks for your comment, @wmheigl. You can provide a topography file with negative elevation to remove that part of the computational domain. However, a free surface boundary condition will still be imposed on the top boundary, which probably is not what you want?

**Comment by wmheigl** [comment]
That is correct, Anders. For my needs I would need absorbing boundaries all around the model.

It’s great to see that you follow up the communities suggestions.
